spec/spec_helper.rb

```diff
@@ -58,26 +58,6 @@
   # Database cleaning
   config.use_transactional_fixtures = true
 
-  config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
-  end
-
-  config.before do
-    DatabaseCleaner.strategy = :transaction
-  end
-
-  config.before(:each, :js) do
-    DatabaseCleaner.strategy = :truncation
-  end
-
-  config.before do
-    DatabaseCleaner.start
-  end
-
-  config.after do
-    DatabaseCleaner.clean
-  end
-
```

Comment:
Are these being removed because of an update to the database cleaner gem?

Reply:
They were actually causing failures & weren't needed anymore because rails 7 improved transactional testing